xla/service/cpu/runtime_fft_impl.h

```diff
@@ -263,6 +263,11 @@ void EigenFftImpl(const EigenDevice& device, void* out, void* operand,
   }
 }
 
+// For now, we will use Eigen thread pools via `EigenDevice`
```

Comment:
Does this apply?  There is no `EigenDevice` here -- though I do suggest adding it.